frontend/app/models/attendance.js

```diff
@@ -20,11 +20,14 @@ export default Model.extend({
 
 , activityDuration: computed('activities.@each.from', 'activities.@each.to', {
     get() {
+      return 0
+      /*
       return moment.duration(this.get('activities').reduce((duration, activity) => {
```

Comment:
I didn't do this because we don't have a proper solution for displaying the activities together with the attendances